DoSomethingArchive/legacy-website
# NOMERGE: Conversion to Drupal VM

#### What's this PR do?

Replaces the current custom Vagrant box with Drupal VM.

#### How should this be reviewed?

1. Check out this branch and run `composer install`.
2. Make sure to remove /html if it exists for you.
3. Run `vagrant up` (note that this might complain about duplicate images with the same name - ping me and we can talk about how best to handle that)
4. Wait a while - it should build the site and clone the staging DB for you.
5. Once it's complete, visit http://dev.dosomething.org and you should see a local site.
6. Also check out http://dashboard.dev.dosomething.org for some other goodies.

#### Any background context you want to provide?

This is VERY new and not thoroughly tested yet, so it'll likely be rough, but it's ready for a first pass.
